tools/gulp/docs/__tests__/createRoutes.test.js

```diff
@@ -1,15 +1,27 @@
 const createRoutes = require('../createRoutes');
 
-function mockPage(slug, sections = []) {
-  return {
+function mockPage(slug, sections = [], customProps = {}) {
+  return Object.assign({
     header: slug,
     description: `${slug} description`,
+    reference: slug,
     referenceURI: slug,
     sections: sections
-  };
+  }, customProps);
 }
```

Comment:
Did you plan on making use of this change? I think it's useful enough to keep, but it doesn't look like it was used.